pyface/ui/qt4/pil_image.py

```diff
@@ -0,0 +1,48 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from pyface.qt.QtGui import QIcon, QPixmap
```

Comment:
Tangent : ~I always wondered why these `pyface.qt` imports preceeded other imports. Why is that?~ Never mind. I think it's because the `pyface.qt` imports are technically Qt-imports which are third party packages and so they should exist before Enthought library imports.

I wish this was codified somewhere because the answer isn't obvious.

Reply:
It's not done universally.  In this case it was likely just copy-pasted from `image_resource.py`.  In other cases as you note it's likely from taking Wx code and translating it and so the Wx imports get replaced by Qt imports in the same place.

It would be nice to standardize, but that would require going through and actually standardizing